lib/rucio/core/replica.py

```diff
@@ -906,7 +906,7 @@ def _list_replicas(dataset_clause, file_clause, state_clause, show_pfns,
                                 # is the RSE site-configured?
                                 rse_site_attr = get_rse_attribute('site', rse_info[rse]['id'], session=session)
                                 replica_site = ['']
-                                if isinstance(rse_site_attr, list) and len(rse_site_attr):
```

Comment:
this actually checks if there is an item in the list, you can't remove the `len` check

Reply:
??
The test is the same
>>> rse_site_attr = []
>>> rse_site_attr is False
False
>>> len(rse_site_attr) is False
False

Reply:
aaargg, the original statement is wrong even that's why i got confused.
it really should be ... `and len(rse_site_attr)>0`

Reply:
???

`if len(rse_site_attr)>0` or `if len(rse_site_attr)` or `if rse_site_attr` should give the same result